var/spack/repos/builtin/packages/openssl/package.py

```diff
@@ -101,7 +101,7 @@ def install(self, spec, prefix):
         # Makefile, leading to build errors.
         env.pop('APPS', None)
 
-        if str(spec.target.family) in ('x86_64', 'ppc64'):
+        if str(spec.target.family) in ('x86_64', 'ppc64', 'arm64'):
```

Comment:
As far as I know `aarch64` and `arm64` are aliases, so I'd rather fix `archspec` detection and classification of the M1 processor, rather than using another name for `aarch64`.

Reply:
See https://github.com/archspec/archspec/issues/38